nestjs/src/courses/mentor-reviews/mentor-reviews.service.ts

```diff
@@ -62,10 +62,16 @@ export class MentorReviewsService {
         'lastChecker.githubId',
       ])
       .addOrderBy('taskSolution.id', 'ASC');
+
+    if (tasks) {
+      const taskIds = tasks.split(',').map(id => parseInt(id, 10));
```

Comment:
10 is a default base, could be just  
```suggestion
      const taskIds = tasks.split(',').map(id => parseInt(id));
```